transport/src/main/java/io/netty5/channel/DefaultChannelPipeline.java

```diff
@@ -1346,4 +1346,6 @@ protected abstract void connectTransport(
      * This method is guaranteed to be called from the {@link #transportExecutor()}.
      */
     protected abstract void sendOutboundEventTransport(Object event, Promise<Void> promise);
+
+    protected abstract boolean isTransportSupportingDisconnect();
```

Comment:
Needs javadocs, I think, since this is a public class.

Reply:
fixed ... PTAL again